### DISCUSSION TITLE
IM GONNA USE A ACTUAL WEBTV BOX FOR WEBTV REDIALED :D

### DISCUSSION THREAD

**Discussion by Selvenity** [SKCro/WebTV-HD#10] [General]
I've used WebTV Viewer for a bit, and i got a WebTV Box. I will watch the VHS tape that's the "introduction to webtv plus" that came inside the box, but when i get a VCR that doesn't eat every tape i insert into it and play.

**Comment by Selvenity** [comment]
However i have to get a VoIP adaptor.

**Comment by SKCro** [comment]
Why not post this info in the WebTV Discord rather than here? I didn't even notice this post until a few minutes ago lol

**Comment by Cinnapoodle** [comment]
> However i have to get a VoIP adaptor.

as someone who connects to WebTV Redialed using VoIP, you don't want to
learn how to make a line voltage inducer or convince your parents to get landline service instead unless you want very slow speeds and constant disconnections

**Reply by tails1154** [reply]
this is so accurate. I used the dial up thing and it took like 1-2 hours to connect lol.
